maintainers/maintainer-list.nix

```diff
@@ -3689,6 +3689,12 @@
     githubId = 343415;
     name = "Greg Roodt";
   };
+  gschwartz = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add gschwartz`` and sort it alphabetically.